packages/components/src/tooltip/test/index.js

```diff
@@ -23,171 +28,136 @@ describe( 'Tooltip', () => {
 	describe( '#render()', () => {
 		it( 'should not render the tooltip if multiple children are passed', () => {
 			render(
-				<Tooltip text="Help text">
-					<button>Button 1</button>
-					<button>Button 2</button>
+				<Tooltip { ...props }>
+					<Button>Button 1</Button>
+					<Button>Button 2</Button>
 				</Tooltip>
 			);
 
-			const button = screen.getByText( 'Button 1' );
-			act( () => button.focus() );
-			expect( screen.queryByText( 'Help text' ) ).not.toBeInTheDocument();
+			expect(
+				screen.queryByText( 'tooltip text' )
+			).not.toBeInTheDocument();
 		} );
 
-		it( 'should render children', () => {
+		it( 'should not render the tooltip if there is no focus', () => {
 			render(
-				<Tooltip text="Help text">
-					<button>Hover Me!</button>
+				<Tooltip { ...props }>
+					<Button>Hover Me!</Button>
 				</Tooltip>
 			);
 
 			expect(
 				screen.getByRole( 'button', { name: 'Hover Me!' } )
 			).toBeVisible();
-			expect( screen.queryByText( 'Help text' ) ).not.toBeInTheDocument();
+			expect(
+				screen.queryByText( 'tooltip text' )
+			).not.toBeInTheDocument();
 		} );
 
-		it( 'should render children with additional tooltip when focused', async () => {
-			const mockOnFocus = jest.fn();
+		it( 'should render the tooltip when focusing anchor via tab', async () => {
+			const user = userEvent.setup();
 
 			render(
-				<Tooltip text="Help text">
-					<button onFocus={ mockOnFocus }>Hover Me!</button>
+				<Tooltip { ...props }>
+					<Button>Hover Me!</Button>
 				</Tooltip>
 			);
 
-			const button = screen.getByRole( 'button', { name: 'Hover Me!' } );
-			expect( button ).toBeVisible();
+			expect( document.body ).toHaveFocus();
```

Comment:
Is that assertion necessary? I'd expect that the document would be focused by default at the beginning of the test anyway?

Reply:
Fair point! I saw this in an example so I thought it was a good first check to do before proceeding, but if it's redundant then I'm happy to remove it. 😄

Reply:
Yeah, AFAIK `document.activeElement` will default to `body` in browsers, but also in jsdom:

https://github.com/jsdom/jsdom/blob/04f6c13f4a4d387c7fc979b8f62c6f68d8a0c639/lib/jsdom/living/nodes/DocumentOrShadowRoot-impl.js#L8